src/rpc/handlers/AccountTx.cpp

```diff
@@ -152,8 +152,15 @@ doAccountTx(Context const& context)
     std::optional<size_t> minReturnedIndex;
     for (auto const& txnPlusMeta : blobs)
     {
-        if (txnPlusMeta.ledgerSequence < minIndex ||
-            txnPlusMeta.ledgerSequence > maxIndex)
+        if (txnPlusMeta.ledgerSequence < minIndex && !forward)
```

Comment:
I think this is wrong. We want to continue if this is entirely out of range. 

I think the change we need to make to fix this is flagging if the returned transactions are out of the range of `minIndex/maxIndex`. 

I'm thinking this will break w/ `forward:true`.